AppKit/CPWindow/CPWindow.j

```diff
@@ -3844,6 +4069,46 @@ var interpolate = function(fromValue, toValue, progress)
 
 @end
 
+@implementation CPWindow (TrackingArea)
+
+- (void)_addToViewsWithTrackingAreas:(CPView)aView
+{
+    [_viewsWithTrackingAreas addObject:aView];
+    
+    var tas = [aView trackingAreas];
+    
+    for (var i = 0, nb = [tas count]; i < nb; i++)
```

Comment:
Please change `nb` to `count`. Even better, don't bother at all, just check `i < trackingAreas.length`. Since Cappuccino arrays will always be bridge with Javascript arrays, we use the native method when possible.